pkg/kubelet/image_manager_test.go

```diff
@@ -32,14 +32,17 @@ import (
 
 var zero time.Time
 
-func newRealImageManager(t *testing.T, policy ImageGCPolicy) (*realImageManager, *dockertools.FakeDockerClient, *cadvisor.Mock) {
+func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *dockertools.FakeDockerClient, *cadvisor.Mock) {
 	fakeDocker := &dockertools.FakeDockerClient{
 		RemovedImages: util.NewStringSet(),
 	}
 	mockCadvisor := new(cadvisor.Mock)
-	im, err := newImageManager(fakeDocker, mockCadvisor, policy)
```

Comment:
I am not sure I understand the downside correctly. If calling `NewMainKubelet()` means you want to start an image manager, that should also applies to garbage collection too?

I don't have strong preference, but would be happy to see them be consistent :)